.github/workflows/release-component.yml

```diff
@@ -55,7 +52,5 @@ jobs:
         env: 
           ENCODED_VALUE: ${{ secrets.SECRING_GPG }}
           OUTPUT_PATH: '.\secring.gpg'
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
       - name: Publish
-        run: ./gradlew $PREVIEW_TASK
+        run: ./gradlew $PREVIEW_TASK -p ${{ inputs.workingDirectory }}
```

Comment:
if I'm following properly this is how the release targets a specific component now?

Reply:
Correct, as mentioned in the PR this is not the final step of the refactoring, it should be possible to release everything with a single command (almost 🙂 ) from the root after those changes.